src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupUtilities.cs

```diff
@@ -33,9 +37,9 @@ public class BackupUtilities
         /// UI input values
         /// </summary>
         private BackupInfo backupInfo;
-        public BackupComponent backupComponent { get; set; }
-        public BackupType backupType { get; set; } // 0 for Full, 1 for Differential, 2 for Log
-        public BackupDeviceType backupDeviceType { get; set; }
+        private BackupComponent backupComponent { get; set; }
```

Comment:
>{ get; set; } [](start = 48, length = 13)

typically private values just use fields, instead of properties.  Is there a reason to use auto-properties here instead of just using fields?